terraform-template-repository.tf.sample

```diff
@@ -43,6 +43,6 @@ resource "null_resource" "example" {
     repo = "${github_repository.example.name}"
   }
   provisioner "local-exec" {
-    command = "initial-commit-tf.sh ${github_repository.example.name}"
+    command = "dataworks-github-config/initial-commit-tf.sh ${github_repository.example.name}"
```

Comment:
the terraform apply command is already running from inside this dir. I'm not sure this is the correct path either. Does it start with the location terraform apply was run from? Would it be better to use the `working_dir` argument?

Reply:
That does sound much better, I'll have a look at that.